Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -328,7 +328,7 @@
   - CHIEF MEDICAL OFFICERS
   - CLOWNS
   - CREW-MEMBERS
-  - CYBORGS
+  - CYBROGS
```

Comment:
Huh?

Reply:
fixed already, wanted to replace it with borgs but changed my mind and spelt it wrong

Reply:
cyfrogs